### ISSUE TITLE
Interest in updated Help?

### ISSUE THREAD

**Issue by hstriepe** [Clozure/ccl#161]
I have created an updated Cocoa-IDE Help for quick access to relevant info. It is probably of greatest use to beginning users.

The style is copied from the classical Help style of Apple apps and provides access to current and legacy docs plus other resources on the web. Since it is indexed, it comes up fairly quickly.

I have attached a screenshot

If there is interest I can add it to a fork.
![ccl help](https://user-images.githubusercontent.com/8574533/46907585-93e7f200-cec9-11e8-8e54-0d3dcf01ba63.png)

**Comment by xrme** [comment]
Yes indeed there is interest.  If I can trouble you to make pull request including this change, I'll merge it in a heartbeat.

**Comment by hstriepe** [comment]
I added a fork - hstriepe/ccl.
Some of the links go back to the old trac wiki. Would be glad to make any needed adjustments. Also updated the Apache license link.

I dropped the old chapter docs from 1.8 and am just using the latest HTML docs with the newer CSS gen'ed from the latest build.

**Comment by xrme** [comment]
This is very interesting.

Are you doing all this by hand, or is there some kind of toolchain involved where you generate style and graphics files?

Especially when it comes to the ccl manual, I think we do not want to check in a ccl.html file to git.  I think the right thing is to build that from the CCLDoc source when we build the bundled app.

**Comment by hstriepe** [comment]
Currently it's by hand. It's mainly the navigation page and index. The latter does not really change much.

It is certainly possible to generate the ccl.html on the fly and then run hiutil to get an index.
